tests/unit/services/intl.js

```diff
@@ -0,0 +1,75 @@
+import { moduleFor } from 'ember-qunit';
+import test from 'dummy/tests/ember-sinon-qunit/test';
+
+let service;
+
+moduleFor('service:intl', 'Unit | Service | intl', {
```

Comment:
I agree, but I don't have time to address that today. Is this sufficient to merge the changes and we can log an issue to provide better overall unit/integration tests for the service?